eq-author-api/constants/answerTypes.js

```diff
@@ -9,6 +9,8 @@ const DATE_RANGE = "DateRange";
 const PERCENTAGE = "Percentage";
 const UNIT = "Unit";
 
+const BASIC_ANSWERS = [TEXTFIELD, TEXTAREA, CURRENCY, NUMBER, UNIT];
```

Comment:
Is this whole file or only this line for testing? If BASIC_ANSWER is only used for testing, maybe give it a comment so it is obvious that the application doesn't depend on it?